ms2/test/src/org/labkey/test/ms2/AbstractMS2SearchEngineTest.java

```diff
@@ -129,9 +134,23 @@ public void basicMS2Check()
 
         clickAndWait(Locator.tagWithAttribute("a", "title", "Experiment run graph"));
 
-        log("Verify msPicture");
+        log("Verify graph view");
         pushLocation();
-        clickAndWait(Locator.imageMapLinkByTitle("graphmap", "Data: " + SAMPLE_BASE_NAME + ".mzXML.image..itms.png (Run Output)"));
+        Locator.linkWithSpan("Toggle Beta Graph (new!)").waitForElement(getDriver(), 4000)
+                .click();
+        LineageGraph graphComponent = new LineageGraph.LineageGraphFinder(getDriver()).waitFor();
+        List<String> actualParents = graphComponent.getDetailGroup("Data Parents")
+                .getItemNames();
+        List<String> actualChildren = graphComponent.getDetailGroup("Data Children")
+                .getItemNames();
+        assertThat(actualParents, hasItems("Bovine_mini1.fasta", "comet.xml", "CAexample_mini.mzXML"));
+        assertThat(actualChildren, hasItems("CAexample_mini.prot.xml", "CAexample_mini.pep.xml",
+                "CAexample_mini.mzXML.image..itms.png"));
+
+        // navigate to the details page for CAexample_mini.mzXML.image..itms.png
+        graphComponent.getDetailGroup("Data Children")
+                .getItem("CAexample_mini.mzXML.image..itms.png")

```

Comment:
Use `SAMPLE_BASE_NAME` where able, to match the rest of the test.
```suggestion
                .getItem(SAMPLE_BASE_NAME + ".mzXML.image..itms.png")
```